src/devices/Directory.Build.props

```diff
@@ -6,5 +6,7 @@
     <DeterministicSourcePaths>false</DeterministicSourcePaths>
     <IsPackable>false</IsPackable>
     <MainLibraryPath>$(MSBuildThisFileDirectory)../System.Device.Gpio/</MainLibraryPath>
```

Comment:
Hey since you're documenting everything in these files, should you also set to true the prop that throws a compiler error when a public API is not documented?

Reply:
This is doing it. By saying you want to generate a documentation file, compiler will throw warnings if public api is not documented. That is why we are setting the second property to treat warnings as errors.